zerver/openapi/zulip.yaml

```diff
@@ -7657,7 +7657,7 @@ paths:
                       email:
                         type: string
                         description: |
-                          Email of the requesting user.
+                          Zulip API email of the requesting user.
```

Comment:
Missed this in the earlier commit, but again an instance where we can clarify what email is returned here.